packages/paima-sdk/paima-utils/src/config/loading.ts

```diff
@@ -147,6 +180,13 @@ const minaConfigDefaults = {
   delay: 30 * 40,
 };
 
+const availConfigDefaults = {
+  funnelBlockGroupSize: 100,
+  confirmationDepth: 10,
+  delay: 10 * 20,
```

Comment:
is there a rationale for this? Theses seem high

Reply:
not really (except for the 20, that's the slot duration), I don't really know what's a sensible default for this. From what I've seen block finalization is usually 3 or 4 blocks behind, but I don't know if that will also be valid for mainnet.

Reply:
yeah finally should be relatively fast even on mainnet. You can ask in the Slack if you want to confirm with the team

Reply:
we could also just not provide any defaults?

Reply:
I think providing defaults makes sense (there is only one Avail chain and I don't imagine there being more than 1), so we just need to provide sensible defaults and a rationale for them

Reply:
I was thinking about that and I started to realize that the `confirmationDepth` setting doesn't really make that much sense here. Avail (and Polkadot) have deterministic finality, and I think the light client doesn't give you blocks that are not finalized anyway. We still need the delay setting to account for this, for which we need a default, but I think I'll just remove this offset

Reply:
Okay, maybe good to mention that in the docs